pkg/sql/importer/import_processor_test.go

```diff
@@ -668,6 +664,7 @@ func TestCSVImportCanBeResumed(t *testing.T) {
 					BulkAdderFlushesEveryBatch: true,
 				},
 			},
+			SQLMemoryPoolSize: 1 << 30, // 1 GiB
```

Comment:
This LGTM, but I think in other places we've dropped how much `kv.bulk_ingest.pk_buffer_size` asks for to begin with to prevent the memory monitor from shouting at us.

Reply:
Done